monetio/models/_cesm_fv_mm.py

```diff
@@ -52,6 +58,31 @@ def open_mfdataset(
     #############################
     # Process the loaded data
     # extract variables of choice
+    # If vertical information is required, add it.
+    # Assume that height and geopotential height are equal
+    if not surf_only:
+        dset_load.rename(
+            {
+                "T": "temperature_k",
+                "Z3": "alt_msl_m_mid",
+                "P0": "surfpres_pa",
+                "PMID": "pres_pa_mid",
+            }
+        )
+        # Calc height agl. PHIS is in m2/s2, whereas Z3 is in already in m
+        dset_load["alt_agl_m_mid"] = dset_load["alt_msl_m_mid"] - dset_load["PHIS"] / 9.81
+        dset_load["alt_agl_m_mid"].attrs = {
+            "description": "geopot height above ground level",
```

Comment:
```suggestion
            "description": "geopotential height above ground level",
```
might as well spell it all out